### DISCUSSION TITLE
쿠링 위젯

### DISCUSSION THREAD

**Discussion by lgvv** [ku-ring/app-ios-v2#116] [Ideas]
Kuring v1, v2에서 모두 사용할 수 있는 widget을 개발할 예정입니다. 

아직 확전된 기획안은 없는데, 차차 생각해보겠습니다

**Comment by lgvv** [comment]
- 대화형 위젯을 통한 구독 상태 변경

**Reply by lgvv** [reply]
공지 카테고리 구독! (앱 실행하지 않고도 위젯으로 구독만하고 푸시알림 받고 ⭐️)

**Reply by x-0o0** [reply]
🔥

**Comment by lgvv** [comment]
- 북마크 공지 리스트

**Comment by lgvv** [comment]
- 쿠링 포토 위젯

**Comment by lgvv** [comment]
- 특정 카테고리 공지 리스트 (상위 5개)?
